django_short_urls/views.py

```diff
@@ -162,6 +162,10 @@ def new(request):  # pylint: disable=too-many-branches
                 status=HTTP_BAD_REQUEST,
                 message="%s may not contain a '/' character." % key)
 
+    statsd.increment(
+        'workforus.new',
+        tags=['prefix:' + unicode(params['prefix']), 'is_secure:' + unicode(request.is_secure())])
```

Comment:
Is the prefix giving us any information regarding / is related to SSL in any way?
It's fine when exploring, but mixing two completely separate information increases the likelihood of tag combinations. Might want two separate metrics here.